README.md

```diff
@@ -0,0 +1,70 @@
+# 2782 Manager
+
+`2782-manager` is a simple controller that allows you to create and update [SRV records](https://datatracker.ietf.org/doc/html/rfc2782), as well as remove them when no endpoints are present.
+
+## About
+
+SRV records are a lightweight service discovery method that can be implemented in a wide range of applications. It uses the well-known DNS protocol to retrieve information about the location, port, and protocol of particular service. `2782-manager` centrally manages the lifecycle of these records.
+
+The initial work will cover a server component and kubernetes controller that watching `Networking/v1 Ingress` objects. Inspired by [external-dns](https://github.com/kubernetes-sigs/external-dns), this is meant to work with an upstream DNS provider, like [ns1](ns1.com).
```

Comment:
what's missing from external-dns? Seems to me like it would be relevant to include the reasons why existing solutions were not sufficient.